.travis.yml

```diff
@@ -82,6 +84,7 @@ jobs:
       - GCP_PROJECT_ID="dynatrace-gcp-extension"
       - GCR_NAME="gcr.io/${GCP_PROJECT_ID}/dynatrace-gcp-function"
       - E2E=true
+      - CLOUD_FUNCTION_NAME="sample_app"
```

Comment:
Please change the function name to something meaningful :) for example e2e_test_sample_app